tiledb/sm/query/readers/sparse_unordered_with_dups_reader.cc

```diff
@@ -1611,11 +1610,9 @@ SparseUnorderedWithDupsReader<BitmapType>::compute_var_size_offsets(
       auto last_tile = (UnorderedWithDupsResultTile<BitmapType>*)
           result_tiles[new_result_tiles_size];
 
-      auto last_tile_num_cells =
-          fragment_metadata[last_tile->frag_idx()]->cell_num(
-              last_tile->tile_idx());
-
       new_result_tiles_size++;
+      auto last_tile_num_cells = cell_offsets[new_result_tiles_size] -
+                                 cell_offsets[new_result_tiles_size - 1];
```

Comment:
Is this called for first tile, is it safe, does it go off beginning of buffer or that allowed for?

Reply:
Since we increment new_result_tiles_size on the previous line, this is safe.